OpenOrchestrator/orchestrator/constants_tab.py

```diff
@@ -131,7 +136,7 @@ def double_click_constant_table(event: tkinter.Event, const_table:ttk.Treeview,
     """
     row = const_table.identify_row(event.y)
     if row:
-        name, value = const_table.item(row, 'values')
+        name, value, _ = const_table.item(row, 'values')
```

Comment:
```suggestion
        name, value, *_ = const_table.item(row, 'values')
```

(cf. https://peps.python.org/pep-3132/)